backend/blueprint/api/utils/response.py

```diff
@@ -9,8 +9,10 @@ def __init__(self, statusCode):
         self.status = statusCode
 
     def generateMessage(self, message):
-        self.message = message
-        return self.serialize()
+        self.id = '1'
```

Comment:
I have assumed it to be one. It has to be changed according to JSON API Conventions. This is just for the working purpose.